rmf_traffic_editor_test_maps/CMakeLists.txt

```
@@ -35,12 +35,21 @@ foreach(path ${traffic_editor_paths})
     )
   else()
     message("DOWNLOADING MODELS WITH COMMAND: ros2 run rmf_building_map_tools building_map_model_downloader ${map_path}")
-    add_custom_command(
-      DEPENDS ${map_path}
-      COMMAND ros2 run rmf_building_map_tools building_map_generator gazebo ${map_path} ${output_world_path} ${output_model_dir}
-      COMMAND ros2 run rmf_building_map_tools building_map_model_downloader ${map_path}
-      OUTPUT ${output_world_path}
-    )
+    if(DEFINED ENV{RMF_DOWNLOAD_MODELS_HOME})
+      add_custom_command(
+        DEPENDS ${map_path}
+        COMMAND ros2 run rmf_building_map_tools building_map_generator gazebo ${map_path} ${output_world_path} ${output_model_dir}
+        COMMAND ros2 run rmf_building_map_tools building_map_model_downloader ${map_path} -f -e ~/.gazebo/models
+        OUTPUT ${output_world_path}
+      )
+    else()
+      add_custom_command(
+        DEPENDS ${map_path}
+        COMMAND ros2 run rmf_building_map_tools building_map_generator gazebo ${map_path} ${output_world_path} ${output_model_dir}
+        COMMAND ros2 run rmf_building_map_tools building_map_model_downloader ${map_path} -f -e ${output_model_dir}
+        OUTPUT ${output_world_path}
+      )
+    endif()
```

```suggestion
    if(DEFINED ENV{RMF_DOWNLOAD_MODELS_HOME})
        set(download_models_path "~/.gazebo/models")
    else()
        set(download_models_path "")
    endif()
    add_custom_command(
        DEPENDS ${map_path}
        COMMAND ros2 run rmf_building_map_tools building_map_generator gazebo ${map_path} ${output_world_path} ${output_model_dir}
        COMMAND ros2 run rmf_building_map_tools building_map_model_downloader ${map_path} -f -e ${download_models_path}
        OUTPUT ${output_world_path}
    )
```

Not tested, could this work?